docs/UserGuide.md

```diff
@@ -430,14 +427,18 @@ Example: `deleteloan 1`
 
 ### Analysing a client's loan records: `analytics`
 
-Provides visual analytics of a client's loan records.
+Provides visual analytics of a client's loan records based on three indices: Reliability, Impact, and Urgency.
+* The Reliability index is defined as the ratio of overdue loans to the total number of loans.
```

Comment:
Do you think we should give examples here? Some users might be unable to visualise the concept of a ratio without an example.

Reply:
I considered this, but I'm worried that it could get pretty cluttered.

Reply:
Hmm that is true, perhaps we could add this in another PR.